src/show/pages/public/Home/index.jsx

```diff
@@ -1,5 +1,5 @@
 const Home = () => {
-  return <div>Home</div>
+  return <div className="flex ">Home</div>
```

Comment:
Remove space after `flex`

Reply:
And everywhere else in the code where you have a space or more before closing or opening the className quotes.